src/main/java/aurora/Duke.java

```diff
@@ -0,0 +1,110 @@
+package aurora;
+
+import aurora.command.ByeCommand;
+import aurora.command.Command;
+import aurora.command.DeadlineCommand;
+import aurora.command.DeleteCommand;
+import aurora.command.EventCommand;
+import aurora.command.FindCommand;
+import aurora.command.InvalidCommand;
+import aurora.command.ListCommand;
+import aurora.command.MarkCommand;
+import aurora.command.TodoCommand;
+import aurora.command.UnmarkCommand;
+import aurora.objects.Deadline;
+import aurora.objects.DukeException;
+import aurora.objects.Event;
+import aurora.objects.Task;
+import aurora.objects.Todo;
+import aurora.parser.Parser;
+import aurora.storage.Storage;
+import aurora.tasklist.TaskList;
+import aurora.ui.Ui;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Duke is the main class for the application that runs according to the commands given to it by the user.
+ */
+public class Duke {
```

Comment:
Just a naming detail. I think that the class name should fit your chatbot's name Aurora, instead of the default "Duke". Similar case for your chatbot's exception class.